lib/bank_accounts/savings_account.rb

```diff
@@ -0,0 +1,52 @@
+module BankAccount
+
+#_________________________________________________________________________
+# SavingsAccount inherits behavior from the Account class
+
+  class SavingsAccount < Account
+
+#__________________________________________________________________________
+# Constant: fee $2
+      FEE = 2
+#__________________________________________________________________________
+# Initialize method
+
+    def initialize(id, initial_balance)
+      super(id, initial_balance)
+
+      # if balance is less than $10 it will raise ArgumentError
+      if @balance < 10
```

Comment:
Same comment as above: this is a good validation, but you're doing it after you've already done work (the `super` call). If the balance is less than 10, there's no reason to run `super()`.